src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs

```diff
@@ -82,61 +68,56 @@ private async void Page_Loaded(object sender, RoutedEventArgs e)
 		{
 			AppSettings.ThemeModeChanged += AppSettings_ThemeModeChanged;
 
-			if (_usingWinUI)
+			if (FilePropertiesHelpers.IsWinUI3)
 			{
-				// WinUI3: Set rectangle for the Titlebar
+				// Set rectangle for the Titlebar
 				TitlebarArea.SizeChanged += (_, _) => DragZoneHelper.SetDragZones(Window, (int)TitlebarArea.ActualHeight);
-				DragZoneHelper.SetDragZones(Window, (int)TitlebarArea.ActualHeight);
-				AppWindow.Destroying += AppWindow_Destroying;
```

Comment:
I think we should keep this as is, I'm pretty sure we had some trouble with Window.Closed.

Reply:
We have a problem with Windows.Destroty and a fixer updated to use Closed so this event will never be fired.